src/transform/destructParam.js

```diff
@@ -25,6 +25,17 @@ export default function(ast, logger) {
               return;
             }
 
+            // skip destructuring, if one of the destructured variable names is already declared by another destruction
+            const duplicateDetected = exs.some(ex => {
+              if (variableNamesCreated.includes(ex.property.name)) {
+                return true;
+              }
+              return false;
+            });
```

Comment:
You can simplify this to
```js
const duplicateDetected = exs.some(ex => {
  return variableNamesCreated.includes(ex.property.name);
});
```
or
```js
const duplicateDetected = exs.some((ex) => variableNamesCreated.includes(ex.property.name));
```
But overall, good job!

Reply:
Good point, fixed!